main.go

```diff
@@ -91,6 +125,20 @@ func seedFixture(n int) []byte {
 	return seed
 }
 
+func getEnv(key, fallback string) string {
+	value := os.Getenv(key)
+	if len(value) > 0 {
+		return value
+	}
+	return fallback
+}
+
+func fatalError(err error) {
+	fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
```suggestion
	fmt.Fprintln(os.Stderr, err)
```

I think the "error: ..." is somewhat redundant, it should be plenty obvious that it is an error :)